src/java.base/share/classes/java/net/SocketPermission.java

```diff
@@ -618,14 +605,13 @@ private boolean isUntrusted()
         if (trusted) return false;
         if (invalid || untrusted) return true;
         try {
-            if (!trustNameService && (defaultDeny ||
-                sun.net.www.URLConnection.isProxiedHost(hostname))) {
```

Comment:
This is important for some internal whitebox test. Please revert this change.

Reply:
> This is important for some internal whitebox test. Please revert this change.

Reverted.